src/db.c

```diff
@@ -668,29 +669,36 @@ serverDb *initTempDb(void) {
     return tempDb;
 }
 
+void freeServerdb(serverDb *db) {
+    if (db == NULL) return;
+    kvstoreRelease(db->keys);
+    kvstoreRelease(db->expires);
+    zfree(db);
+}
+
```

Comment:
This seems strange and incomplete. if we are writing a dedicated function to free a DB we need to make sure ALL it's fields are being reclaimed (or assert if they are in use). What about the watched keys? blocked keys? etc...

Reply:
This has been removed.